mlir/lib/Dialect/MIOpen/Generator/CMakeLists.txt

```diff
@@ -37,3 +41,7 @@ target_link_libraries(MLIRMIOpenConv2dGenerator
   MLIRMIOpenOps
   ${LIBS}
 )
+
+add_llvm_library(MLIRRocmBackendUtils
```

Comment:
add_llvm_library is absolutely not the function you want - that has the implication that the component is part of LLVM, not MLIR. `add_mlir_dialect_library`, maybe?

Reply:
According to the definition of mlir_add_dialect_library, this function declares a library associated with a dialect. Should we instead use the more general function mlir_add_library for IsaNamePaser since it is more like a utility function?

Reply:
This is a library related to the `MIOpen` dialect, given which folder it's in